cns/restserver/ipam.go

```diff
@@ -26,7 +35,13 @@ var (
 
 // requestIPConfigHandlerHelper validates the request, assign IPs and return the IPConfigs
 func (service *HTTPRestService) requestIPConfigHandlerHelper(ctx context.Context, ipconfigsRequest cns.IPConfigsRequest) (*cns.IPConfigsResponse, error) {
-	// For SWIFT v2 scenario, the validator function will also modify the ipconfigsRequest.
+	cmdLineConfigPath := common.GetArg(common.OptCNSConfigPath).(string)
+	cnsconfig, err := configuration.ReadConfig(cmdLineConfigPath)
+	var swiftv2sf bool
```

Comment:
can we add some comments here to explain more about swiftv2sf scenario, for example how we can get podInfo with this flag; also I remember we will need to support the K8S, so we will still use this flag I believe?

Reply:
added